--- a/src/pulsecore/core-util.c	2024-01-29 17:39:02.607045839 +0000
+++ b/src/pulsecore/core-util.c	2024-01-29 17:36:36.787635272 +0000
@@ -1767,7 +1767,7 @@
      * set create a directory in $HOME and link it to a random subdir
      * in /tmp, if it was not explicitly configured. */
 
-    m = pa_in_system_mode() ? 0755U : 0700U;
+    m = pa_in_system_mode() ? 0755U : 0710U;
 
     /* Use the explicitly configured value if it is set */
     d = getenv("PULSE_RUNTIME_PATH");
